github/orgs_issue_types.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2025 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// CreateOrUpdateIssueTypesOptions represents the parameters for creating or updating an issue type.
+type CreateOrUpdateIssueTypesOptions struct {
+	Name        *string `json:"name"`                  // Name of the issue type. (Required.)
+	IsEnabled   *bool   `json:"is_enabled"`            // Whether or not the issue type is enabled at the organization level. (Required.)
```

Comment:
Sorry I didn't catch this before, but let's please make these two required fields values instead of pointers.

```suggestion
	Name        string `json:"name"`                  // Name of the issue type. (Required.)
	IsEnabled   bool   `json:"is_enabled"`            // Whether or not the issue type is enabled at the organization level. (Required.)
```

Reply:
> Sorry I didn't catch this before, but let's please make these two required fields values instead of pointers.

Thanks @gmlewis , I've updated those required fields to values. I'm still learning and I'd like to understand why we prefer values instead of pointers for required fields. could you clarify the reasoning behind this choice?

Reply:
Oh, sure!  There have been several discussions about this topic in this repo before, but I'll give you a quick summary. We use pointers and `omitempty` for optional fields so that they are not included in the JSON when Go marshals the structs to JSON. When they are required, as in this case, we prefer to use values so that it is obvious to the user of this client library that they need to provide a value (and can't leave it as `nil` if it were a pointer).
I hope that makes sense. Please let me know if you have further questions, and I can try to dig up older conversations that go into more details.

Reply:
Got it now! Thanks a lot for the clear explanation @gmlewis Really appreciate it!